docs/development/how_to_add_a_route.md

```diff
@@ -0,0 +1,175 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+The goal of this document is to cover all the necessary steps to add an API route to Parsec.
```

Comment:
Instead of a"API route", I'd would talk about "API RPC command" given we have `invitated_cmds`/`authenticated_cmds`/`anonymous_cmds` modules that contains the schemas of those RPC commands (and "API route" may seem to refer to the administration REST API which is a totally unrelated component).

Reply:
so I guess this file should be renamed to something like `how_to_add_a_rpc_cmd.md`